var/spack/repos/builtin/packages/functionalizer/package.py

```diff
@@ -0,0 +1,75 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Functionalizer(CMakePackage):
+    """Apply several steps of filtering on touches
+    """
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/building/Functionalizer"
+    url      = "ssh://bbpcode.epfl.ch/building/Functionalizer"
+
+    version('develop',
+            git=url,
+            submodules=True)
+    version('3.11.0',
+            commit='50c83265c100cec66a27eea9311b58a9b652cb5f',
+            git=url,
+            preferred=True,
+            submodules=True)
+    version('gap-junctions',
+            commit='6095a851119d8125a81f2858c7a0de2ff6f012d6',
+            git=url,
+            submodules=True)
+
+    depends_on('boost@1.50:')
+    depends_on('cmake', type='build')
+    depends_on('cmake@:3.0.0', type='build', when='@gap-junctions')
```

Comment:
this is restriction cmake < 3.0. If I look into source code:

```
cmake_minimum_required(VERSION 2.8)
```

So we don't need above line?

Reply:
Building fails due to new CMake idioms. PR updated with the other comments.